kernel/src/vtpm/tcgtpm/tss.rs

```diff
@@ -132,10 +137,10 @@ pub fn create_ek<T: TcgTpmSimulatorInterface>(
 ) -> Result<Vec<u8>, SvsmVTpmError> {
     let mut cmd = create_mtauth_ek_cmd(tpmt_public);
 
-    checked_send(vtpm, &mut cmd, /*set_len=*/ false)?;
+    let mut response = checked_send(vtpm, &mut cmd, /*set_len=*/ false)?;
 
     // Get size (UINT16) of TPMT_PUBLIC at offset 18.
     // Note this is output from the TPM, so its value is trusted.
-    let size_of_tpmt_public = u16::from_be_bytes([cmd[18], cmd[19]]);
-    Ok(cmd[20..(20 + size_of_tpmt_public) as usize].to_vec())
+    let size_of_tpmt_public: usize = u16::from_be_bytes([response[18], response[19]]).into();
```

Comment:
```suggestion
    let size_of_tpmt_public = u16::from_be_bytes([response[18], response[19]]) as usize;
```
Just for consistency of how we do this elsewhere.